parallel-lint.php

```diff
@@ -92,7 +93,7 @@ function showUsage()
 try {
     $settings = PhpParallelLint\Settings::parseArguments($_SERVER['argv']);
 
-    if ($settings->json && PHP_VERSION_ID < 50400) {
+    if (PhpParallelLint\Settings::FORMAT_JSON === $settings->format && PHP_VERSION_ID < 50400) {
```

Comment:
Can you please change the direction of condition?

Reply:
PHP<5.4 is out of question so it's just going to be dropped.